### PR TITLE
Add support for different machine image and version in tm tests

### PR DESCRIPTION
**Pull request by schrodit**
**What this PR does / why we need it**:
Adds support to specify other machine images and versions to support suse jeos tests

**Special notes for your reviewer**:
cc @marwinski 
**Release note**:

```improvement operator
Add support for different machine image and version in tm tests
```

### CODE REVIEW

File: .test-defs/cmd/helper.go [hunk 1]
```diff
@@ -43,6 +43,31 @@ func UpdateWorkerZone(shoot *gardenv1beta1.Shoot, cloudprovider gardenv1beta1.Cl
...
+		case gardenv1beta1.CloudProviderAWS:
+			shoot.Spec.Cloud.AWS.MachineImage = machineImage
+		case gardenv1beta1.CloudProviderGCP:
+			shoot.Spec.Cloud.GCP.MachineImage = machineImage
+		case gardenv1beta1.CloudProviderAzure:
+			shoot.Spec.Cloud.Azure.MachineImage = machineImage
+		case gardenv1beta1.CloudProviderOpenStack:
+			shoot.Spec.Cloud.OpenStack.MachineImage = machineImage
+		case gardenv1beta1.CloudProviderAlicloud:
+			shoot.Spec.Cloud.Alicloud.MachineImage = machineImage
```
Comment (rfranzke) [inline .test-defs/cmd/helper.go:65]: Packet is missing

Comment (schrodit) [reply .test-defs/cmd/helper.go:65]: Also added packet to the other update functions

Comment (rfranzke) [reply .test-defs/cmd/helper.go:65]: Great

**Review by rfranzke — APPROVED** [review]
/lgtm

**Pull request merged**